bench/src/lib.rs

```diff
@@ -160,10 +161,16 @@ pub struct Opt {
     #[structopt(long = "max_streams", short = "m", default_value = "1")]
     pub max_streams: usize,
     /// Number of bytes to transmit from server to client
-    #[structopt(long, default_value = "1073741824")]
+    ///
+    /// This can use SI prefixes for sizes. E.g. 1M will transfer 1MB, 10G
+    /// will transfer 10GB.
```

Comment:
Should say `1MiB` and `10GiB`, and similar for `upload_size`.

Reply:
good call. I actually also don't like the ambiguous units. Updated